src/acl.c

```diff
@@ -2066,43 +2412,17 @@ void aclCommand(client *c) {
             sds thispass = listNodeValue(ln);
             addReplyBulkCBuffer(c,thispass,sdslen(thispass));
         }
+        /* Include the default selector at thee top level for backwards compatibility */
+        aclGetUserAddSelectorReply(c, ACLUserGetDefaultSelector(u));
 
-        /* Commands */
-        addReplyBulkCString(c,"commands");
-        sds cmddescr = ACLDescribeUserCommandRules(u);
-        addReplyBulkSds(c,cmddescr);
-
-        /* Key patterns */
-        addReplyBulkCString(c,"keys");
-        if (u->flags & USER_FLAG_ALLKEYS) {
-            addReplyArrayLen(c,1);
-            addReplyBulkCBuffer(c,"*",1);
-        } else {
-            addReplyArrayLen(c,listLength(u->patterns));
-            listIter li;
-            listNode *ln;
-            listRewind(u->patterns,&li);
-            while((ln = listNext(&li))) {
-                sds thispat = listNodeValue(ln);
-                addReplyBulkCBuffer(c,thispat,sdslen(thispat));
-            }
-        }
-
-        /* Pub/sub patterns */
-        addReplyBulkCString(c,"channels");
-        if (u->flags & USER_FLAG_ALLCHANNELS) {
-            addReplyArrayLen(c,1);
-            addReplyBulkCBuffer(c,"*",1);
-        } else {
-            addReplyArrayLen(c,listLength(u->channels));
-            listIter li;
-            listNode *ln;
-            listRewind(u->channels,&li);
-            while((ln = listNext(&li))) {
-                sds thispat = listNodeValue(ln);
-                addReplyBulkCBuffer(c,thispat,sdslen(thispat));
-            }
-        }
+        /* Describe all of the selectors on this user, including duplicating the default selector */
+        addReplyBulkCString(c,"selectors");
+        addReplyArrayLen(c, listLength(u->selectors));
+        listRewind(u->selectors,&li);
```

Comment:
please describe the interface change to GETUSER and any others in the top comment of the PR (easier review and release notes)